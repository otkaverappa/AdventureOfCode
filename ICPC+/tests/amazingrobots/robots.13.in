20 20
####################
#..................#
#..................#
#..................#
#...X..............#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#...................
#..................#
####################
2
19 19 2 N
18 18 3 N
20 20
####################
#.....#...#...#....#
....#.#.#.#.#.#.#..#
#.###...#...#...#..#
#...#############..#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#.................X#
####################
1
2 2 2 S
