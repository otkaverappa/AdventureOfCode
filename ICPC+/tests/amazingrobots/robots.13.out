66
E
E
E
E
E
E
E
E
E
E
E
E
E
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
W
W
N
N
W
W
